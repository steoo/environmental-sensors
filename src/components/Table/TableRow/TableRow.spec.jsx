import React from 'react';
import { render } from '@testing-library/react';
import { Table, TableBody } from '@material-ui/core';
import ResultTableRow from './TableRow.component';

const tableRowData = {
  id: 'Box-A2-CO',
  box_id: 'Box-A2',
  sensor_type: 'CO',
  unit: 'ppm',
  name: 'Carbon monoxide',
  reading: 437,
  reading_ts: '2019-09-10T00:30:00',
};

describe('Components', () => {
  describe('Table Row', () => {
    it('should render, without crashing', () => {
      const { container } = render(
        <Table>
          <TableBody>
            <ResultTableRow {...tableRowData} />
          </TableBody>
        </Table>,
      );

      expect(container).toMatchSnapshot();
    });
  });
});
