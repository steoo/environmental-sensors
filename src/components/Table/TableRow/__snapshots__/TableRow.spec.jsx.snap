// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Table Row should render, without crashing 1`] = `
<div>
  <table
    class="MuiTable-root"
  >
    <tbody
      class="MuiTableBody-root"
    >
      <tr
        class="MuiTableRow-root"
      >
        <td
          class="MuiTableCell-root MuiTableCell-body"
        >
          Box-A2-CO
        </td>
        <td
          class="MuiTableCell-root MuiTableCell-body"
        >
          Box-A2
        </td>
        <td
          class="MuiTableCell-root MuiTableCell-body"
        >
          CO
        </td>
        <td
          class="MuiTableCell-root MuiTableCell-body"
        >
          Carbon monoxide
        </td>
        <td
          class="MuiTableCell-root MuiTableCell-body"
        >
          ppm
        </td>
        <td
          class="MuiTableCell-root MuiTableCell-body"
        >
          437
        </td>
        <td
          class="MuiTableCell-root MuiTableCell-body"
        >
          2019-09-10T00:30:00
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
